scripts/_common.py

```diff
@@ -31,10 +32,21 @@
     PROJECT_DIR, "kolibri_explore_plugin", "static", "highlighted-content.json"
 )
 
+BUNDLE_EXCLUDE_GLOBS = ["*.map"]
 
-def bundle_zip(zip_name):
-    shutil.make_archive(zip_name, "zip", "dist/")
-    print(f"File ./{zip_name}.zip created.")
+
+def bundle_zip(zip_name, compression=zipfile.ZIP_LZMA):
```

Comment:
This breaks the Explore Plugin in the my chromebook build: `RuntimeError: Compression requires the (missing) lzma module`

Reply:
Ah, thanks for testing that :) I updated endlessm/kolibri-installer-android#2 to include `liblzma` (https://github.com/kivy/python-for-android/blob/master/pythonforandroid/recipes/liblzma/__init__.py) in p4a's requirements list, which should solve that issue.